google/cloud/aiplatform/training_jobs.py

```diff
@@ -2584,6 +2594,17 @@ def __init__(
                 If an input column has no transformations on it, such a column is
                 ignored by the training, except for the targetColumn, which should have
                 no transformations defined on.
+                Only one of column_transformations or column_specs should be passed.
+            column_transformations (Union[Dict, List[Dict]]):
+                Optional. Transformations to apply to the input columns (i.e. columns other
+                than the targetColumn). Each transformation may produce multiple
+                result values from the column's value, and all are used for training.
+                When creating transformation for BigQuery Struct column, the column
```

Comment:
You also might want to note that only columns with no child columns of their own should have a transformation applied.

For example, if you have a column "field_a.field_b", then pass in the transformation for "field_a.field_b" but do not pass in the transformation for "field_a".